### PR TITLE
removed & added Serializable where needed

### PR DESCRIPTION
**Pull request by theo-ritense**
Also:
- import cleanups
- other SonarQube findings

### CODE REVIEW

File: contract/src/main/java/com/ritense/valtimo/contract/documentgeneration/event/DossierDocumentGeneratedEvent.java [hunk 1]
```diff
@@ -19,11 +19,10 @@
...
 import com.ritense.valtimo.contract.audit.AuditEvent;
 import com.ritense.valtimo.contract.audit.AuditMetaData;
-import java.io.Serializable;
 import java.time.LocalDateTime;
 import java.util.UUID;
 
-public class DossierDocumentGeneratedEvent extends AuditMetaData implements AuditEvent, Serializable {
+public class DossierDocumentGeneratedEvent extends AuditMetaData implements AuditEvent {
 
     private static final long serialVersionUID = -1001787410861765142L;
```
Comment (Klaas-Ritense) [inline contract/src/main/java/com/ritense/valtimo/contract/documentgeneration/event/DossierDocumentGeneratedEvent.java:27]: Could be removed

File: process-document/src/main/java/com/ritense/processdocument/domain/impl/CamundaProcessDefinitionKey.java [hunk 2]
```diff
@@ -19,12 +19,13 @@
...
 import javax.persistence.Column;
 import javax.persistence.Embeddable;
+import java.io.Serializable;
 import java.util.Objects;
 import static com.ritense.valtimo.contract.utils.AssertionConcern.assertArgumentLength;
 import static com.ritense.valtimo.contract.utils.AssertionConcern.assertArgumentNotNull;
 
 @Embeddable
-public class CamundaProcessDefinitionKey implements ProcessDefinitionKey {
+public class CamundaProcessDefinitionKey implements ProcessDefinitionKey, Serializable {
```
Comment (Klaas-Ritense) [inline process-document/src/main/java/com/ritense/processdocument/domain/impl/CamundaProcessDefinitionKey.java:28]: Is `Serializable` need here?

File: process-document/src/main/java/com/ritense/processdocument/domain/impl/CamundaProcessInstanceId.java [hunk 3]
```diff
@@ -19,14 +19,15 @@
...
+import java.io.Serializable;
 import java.util.Objects;
 import java.util.UUID;
 import static com.ritense.valtimo.contract.utils.AssertionConcern.assertArgumentLength;
 import static com.ritense.valtimo.contract.utils.AssertionConcern.assertArgumentNotNull;
 import static com.ritense.valtimo.contract.utils.AssertionConcern.assertArgumentTrue;
 
 @Embeddable
-public class CamundaProcessInstanceId implements ProcessInstanceId {
+public class CamundaProcessInstanceId implements ProcessInstanceId, Serializable {
```
Comment (Klaas-Ritense) [inline process-document/src/main/java/com/ritense/processdocument/domain/impl/CamundaProcessInstanceId.java:30]: Is `Serializable` needed here?

**Review by Klaas-Ritense — APPROVED** [review]

File: contract/src/main/java/com/ritense/valtimo/contract/documentgeneration/event/DossierDocumentGeneratedEvent.java (hunk 1 above)
Comment (theo-ritense) [reply contract/src/main/java/com/ritense/valtimo/contract/documentgeneration/event/DossierDocumentGeneratedEvent.java]: Done

File: process-document/src/main/java/com/ritense/processdocument/domain/impl/CamundaProcessDefinitionKey.java (hunk 2 above)
Comment (theo-ritense) [reply process-document/src/main/java/com/ritense/processdocument/domain/impl/CamundaProcessDefinitionKey.java:28]: Yes, the JPA spec required Id's to be serializable. This key is part of one.

File: process-document/src/main/java/com/ritense/processdocument/domain/impl/CamundaProcessInstanceId.java (hunk 3 above)
Comment (theo-ritense) [reply process-document/src/main/java/com/ritense/processdocument/domain/impl/CamundaProcessInstanceId.java:30]: Yes, the JPA spec requires Id's to be serializable.

**Pull request merged**